pkg/ipamd/ipamd.go

```diff
@@ -225,16 +225,19 @@ func (c *IPAMContext) setUnmanagedENIs(tagMap map[string]awsutils.TagMap) {
 	if len(tagMap) == 0 {
 		return
 	}
+	var unmanagedENIlist []string
 	for eniID, tags := range tagMap {
 		if tags[eniNoManageTagKey] == "true" {
 			if eniID == c.awsClient.GetPrimaryENI() {
 				log.Debugf("Ignoring no_manage tag on primary ENI %s", eniID)
 			} else {
 				log.Debugf("Marking ENI %s tagged with %s as being unmanaged", eniID, eniNoManageTagKey)
 				c.unmanagedENIs.add(eniID)
+				unmanagedENIlist = append(unmanagedENIlist, eniID)
 			}
 		}
 	}
+	c.awsClient.SetUnmanagedENIs(unmanagedENIlist)
```

Comment:
Instead of having this set in both `ipamd.go` and `awsutils.go`, can't we just move it to `awsutils.go`?

Reply:
You mean we can remove it from ipamd context?

Reply:
Yes, just keep one set around.

Reply:
sure, will make that change and resubmit PR :)